docs/source/tts/configs.rst

```diff
@@ -0,0 +1,263 @@
+
+.. code-block:: yaml
+
+  model:
+    train_ds:
+      dataset:
+        _target_: nemo.collections.tts.torch.data.TTSDataset
+        manifest_filepath: ???
+        sample_rate: 44100
+        sup_data_path: ???
+        sup_data_types: ["align_prior_matrix", "pitch"]
+        n_fft: 2048
+        win_length: 2048
+        hop_length: 512
+        window: hann
+        n_mels: 80
+        lowfreq: 0
+        highfreq: null
+        max_duration: null
+        min_duration: 0.1
+        ignore_file: null
+        trim: false
+        pitch_fmin: 65.40639132514966
+        pitch_fmax: 2093.004522404789
+        pitch_norm: true
+        pitch_mean: 212.35873413085938
+        pitch_std: 68.52806091308594
+        use_beta_binomial_interpolator: true
+
+      dataloader_params:
+        drop_last: false
+        shuffle: true
+        batch_size: 32
+        num_workers: 12
+        pin_memory: true
+
+
+Audio Preprocessor Configuration
+--------------------------------
+
+If you are loading audio files for your experiment, you will likely want to use a preprocessor to convert from the raw audio signal to features (e.g. mel-spectrogram or MFCC). The ``preprocessor`` section of the config specifies the audio preprocessor to be used via the ``_target_`` field, as well as any initialization parameters for that preprocessor. An example of specifying a preprocessor is as follows. Refer to the `Audio Preprocessors <../asr/api.html#Audio Preprocessors>`__ API section for the preprocessor options, expected arguments, and defaults.
+
+.. code-block:: yaml
+
+  model:
+    preprocessor:
+      _target_: nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor
+      features: 80
+      lowfreq: 0
+      highfreq: null
+      n_fft: 2048
+      n_window_size: 2048
+      window_size: false
+      n_window_stride: 512
+      window_stride: false
+      pad_to: 1
+      pad_value: 0
+      sample_rate: 44100
+      window: hann
+      normalize: null
+      preemph: null
+      dither: 0.0
+      frame_splicing: 1
+      log: true
+      log_zero_guard_type: add
+      log_zero_guard_value: 1e-05
+      mag_power: 1.0
+
+Text Normalizer Configuration
+------------------------------
+Text normalization (TN) converts text from written form into its verbalized form, and it is an essential preprocessing step before text-to-speech Synthesis. TN ensures that TTS can handle all input texts without skipping unknown symbols. For example, "$123" is converted to "one hundred and twenty-three dollars". Currently, NeMo supports text normalizers for English, Russian, German, Spanish, and Chinese. Refer to the previous Section :doc:`../nlp/text_normalization/intro` for more details. Below shows an example of specifying text normalizer for English.
```

Comment:
Please remove Russian, it's audio-based only

Reply:
"twenty-three" -> "twenty three". What do you mean by "previous Section"? remove "previous"?

Reply:
TN WFST docs are missing latest installation instructions, do you mind adding `wget https://github.com/NVIDIA/NeMo/blob/main/nemo_text_processing/install_pynini.sh && bash install_pynini.sh` to [this doc](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/text_normalization/wfst/wfst_text_normalization.html#installation)?

Reply:
fixed.